### ISSUE TITLE
wildcard usage at CLASSPATH breaks stop.bat and start.bat

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#16355]
`stop.bat` file does not stop hazelcast instance due to `;%~dp0..\user-lib\*` path at CLASSPATH env variable even if I use same CLASSPATH variable with `start.bat` at `stop.bat`:
https://github.com/hazelcast/hazelcast/blob/maintenance-3.x/hazelcast/src/main/resources/start.bat#L26
https://github.com/hazelcast/hazelcast/blob/maintenance-3.x/hazelcast/src/main/resources/stop.bat#L5

Somehow `;%~dp0..\user-lib\*` section breaks `WINDOWTITLE eq hazelcast %CLASSPATH%` checks at `start.bat` and `stop.bat`, I could not find any issue or doc about it:
https://github.com/hazelcast/hazelcast/blob/v3.12.5/hazelcast/src/main/resources/start.bat#L28-L33

@7erry reported the another issue related to title at `start` command at `start.bat`:
```
C:\..\3.12.5\bin>start.bat
########################################

RUN_JAVA="c:\Program Files (x86)\Java\jre1.8.0_231"\bin\java
JAVA_OPTS=
starting now...."
########################################
The system cannot accept the START command parameter ......
----
the following modified start.bat appears to work:
start "hazelcast" %RUN_JAVA% %JAVA_OPTS% -cp %CLASSPATH% "com.hazelcast.core.server.StartServer"
```
https://github.com/hazelcast/hazelcast/blob/maintenance-3.x/hazelcast/src/main/resources/start.bat#L41

**Comment by hasancelik** [comment]
similar issue --> https://github.com/hazelcast/hazelcast/issues/14217

**Comment by hasancelik** [comment]
fixed by #16356
